src/codemagic/tools/_app_store_connect/action_groups/builds_action_group.py

```diff
@@ -531,13 +543,16 @@ def _get_existing_app_store_version_localizations(
         }
 
     def _get_editable_app_store_version(self, app: App, platform: Platform) -> Optional[AppStoreVersion]:
+        def sorting_key(app_store_version: Optional[AppStoreVersion]) -> versions.Version:
+            assert app_store_version is not None  # Make mypy happy
+            return versions.sorting_key(app_store_version.attributes.versionString)
+
         versions_filter = self.api_client.app_store_versions.Filter(
             app_store_state=AppStoreState.editable_states(),
             platform=platform,
         )
         app_store_versions = self.api_client.apps.list_app_store_versions(app, resource_filter=versions_filter)
-        app_store_versions.sort(key=lambda asv: LooseVersion(asv.attributes.versionString))
-        return app_store_versions[-1] if app_store_versions else None
+        return max(app_store_versions, default=None, key=sorting_key)
```

Comment:
This is the only relevant change in this file except the new import at the top. Everything else is autoformatting to be compliant with `flake8` rules.